src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -55,7 +64,7 @@ public static void openWranglerConnectionsPage() throws IOException {
    * Click on the Browser connections button inside Plugin's properties page
    */
   public static void clickBrowseConnectionsButton() {
-    ElementHelper.clickOnElement(CdfConnectionLocators.browseConnectionsButton);
+    clickOnElement(CdfConnectionLocators.browseConnectionsButton);
```

Comment:
Once you remove the method import from above revert this change and use ```ElementHelper.clickOnElement()``` in all the methods.

Reply:
resolved